packages/compass-global-writes/src/plugin-title.tsx

```diff
@@ -49,7 +55,7 @@ export const PluginTitle = ({ showWarning }: { showWarning: boolean }) => {
               }}
             >
               <Icon
-                glyph="ImportantWithCircle"
+                glyph={showWarning ? 'Warning' : 'ImportantWithCircle'}
```

Comment:
The other way around :) I know it's confusing, but we want it to match the banner icon
<img width="722" alt="Screenshot 2024-11-12 at 18 10 07" src="https://github.com/user-attachments/assets/a8276861-de42-4332-ab08-7115018ef227">
<img width="727" alt="Screenshot 2024-11-12 at 18 15 34" src="https://github.com/user-attachments/assets/43381a96-e4bc-4d1c-8e43-181b4193aa38">